### DISCUSSION TITLE
.Net Framewotk project can update to version 3.1.5

### DISCUSSION THREAD

**Discussion by berndrausch** [SixLabors/ImageSharp#2826] [Q&A] [answered]
I have a Visual Studio project based on .Net framework 4.8 that refernences SixLabors.Fonts in version 1.0.1 and SixLabors.ImageSharp in version 2.1.9. If I try to update SixLabors.Fonts to 2.0.4 using "Manage nuget packages for solution", I get an error during the update, because .NetFramework is not supported in SixLabors.Fonts 2.0.4. This is fine and I respect this decision. But when I try to update SixLabors.ImageSharp to version 3.1.5, I do not get any error during the update. But in my understanding, SixLabors.ImageSharp 3.1.5 also does not support .Net Framework. 
My question is how to handle this? Is it considered a bug in SixFlags.ImageSharp?
To be clear: I do not expect that newer versions of SixFlags.ImageSharp support .Net Framework, I just expect an error when I try to update it (just like I get an error when I try to update SixLabors.Fonts).

**Comment by JimBobSquarePants** [comment, marked as answer]
I think you're just not seeing the error in Visual Studio. 

The supported framework collection is identical for both libraries. 

![image](https://github.com/user-attachments/assets/6b219e84-eeef-4553-bad2-802005cf372c)

https://www.nuget.org/packages/SixLabors.ImageSharp/#supportedframeworks-body-tab


![image](https://github.com/user-attachments/assets/47bbcdef-013c-4672-9947-d2d0d7dec888)

https://www.nuget.org/packages/SixLabors.Fonts#supportedframeworks-body-tab

If you are continuing to not see a warning I would raise that with the NuGet team.
